program/rust/src/tests/test_add_price.rs

```diff
@@ -62,6 +63,9 @@ fn test_add_price() {
     let mut permissions_setup = AccountSetup::new_permission(&program_id);
     let permissions_account = permissions_setup.as_account_info();
 
+    let mut scores_setup = AccountSetup::new::<PublisherScoresAccount>(&program_id);
+    let scores_account = scores_setup.as_account_info();
```

Comment:
There will need to be a way to initialize this account in the program:
1. If the account is the PDA (like the permssions account) you're limited to 10kb at creation but you can resize it later (increase it by 10kb) at a time
2. If it's a regular account the limit is 10MB. The problem is the address will be at a random address (instead of deterministically derived from a seed).

Reply:
we probably will go with number 2. We can also hardcode the account to avoid duplication.